core/src/main/java/cucumber/runner/TimeService.java

```diff
@@ -10,20 +10,4 @@ public long time() {
         }
     };
 
-    class Stub implements TimeService {
-        private final long duration;
```

Comment:
I've moved this class into the test packages. Doesn't belong in our public code.